server/lib/notifications.js

```diff
@@ -137,6 +140,49 @@ export async function notifyAdminsOfCollective(CollectiveId, activity, options =
   return notifySubscribers(adminUsers, activity, options);
 }
 
+async function notifyUserIfExpensesExceededMinimumFormAmount(activity) {
```

Comment:
this is a very long function. I'd split it by creating a helper function attached to the `Expense` model:

```js
models.Expense.getTotalExpensesFromUserId = async function(UserId, since, until = new Date) {
  // ...
}
```

```js
async function w9bot(activity) {
  const hostCollective = await models.Collective.findById(activity.data.host.id);
  if (hostCollective.currency !== 'USD') return;
  // ideally change to consider the current fiscal year instead(Example: might be from
  // March to February instead of from the beginning of January)
  const beginningOfCurrentYear = new Date(new Date().getFullYear(), 0, 1);
  const totalAmountThisYear = await models.Expense.getTotalExpensesFromUserId(UserId, beginningOfCurrentYear);
  if (totalAmountThisYear > 60000) {
    // create comment
  }
}
```